.papr.yml

```diff
@@ -167,3 +167,26 @@ tests:
 
 artifacts:
   - ex-container-logs
+
+---
+
+branches:
+  - master
+  - auto
+  - try
+  - pr/rust-check
+
+context: rust-min-version-check
+timeout: 30m
+
+required: true
+container:
+  image: registry.fedoraproject.org/fedora:28
+
+env:
+  RUST_MIN_VERSION: 1.26.2
+
+tests:
+  - curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain $RUST_MIN_VERSION -y
+  - PATH=$HOME/.cargo/bin:$PATH ci/build.sh | tee out.log
```

Comment:
Do we need to build everything here?  Couldn't we just build the Rust code?

Reply:
Sure, fixed below! We do still need some of the build deps installed by `build.sh` though like glib, libcurl, etc... So I've split out a separate `installdeps.sh` script.